### PR TITLE
Update tea notebook with rotator plots

### PR DESCRIPTION
**Pull request by b1quint**

### CODE REVIEW

File: notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1285_top_end_assembly_vibration.ipynb [hunk 1]
```diff
@@ -109,9 +109,9 @@
     "# start = Time(\"2024-03-10T18:00:00Z\", scale=\"utc\", format=\"isot\")\n",
```
Comment (boutigny) [inline notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1285_top_end_assembly_vibration.ipynb:109]: <p>That's a detail, but for me it takes 4.5 min</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/140/discussion/#comment-1684967907'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (boutigny) [inline notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1285_top_end_assembly_vibration.ipynb:109]: <pre class="ql-syntax" spellcheck="false">Line #6.        long_name="SST M2 Surrogate",
</pre><p>Trivial, but what is the meaning of SST ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/140/discussion/#comment-1684967908'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

**Comment by boutigny** [comment]
On top of the above very minor comments, I would suggest to have the same scales for rotator on and rotator off plots.
I also see that the plot corresponding to "BLOCK-197 - Hexapod Analysis" have no axis and no titles.

Otherwise everything looks good. Feel free to merge

**Review by boutigny — APPROVED** [review]
All comments are minor ones.

File: notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1285_top_end_assembly_vibration.ipynb (hunk 1 above)
Comment (b1quint) [reply notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1285_top_end_assembly_vibration.ipynb:109]: I updated the message to warn about this.

Comment (b1quint) [reply notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1285_top_end_assembly_vibration.ipynb:109]: I added a comment to answer this question. SST stands for Simonyi Survey Telescope.

**Comment by b1quint** [comment]
@boutigny I merged the PR you reviewed. I addressed some of the comments. The only comment I did not address was about the scales when comparing rotator on/off. That would be way too complicated.

**Pull request merged**